eppoclient/client.go

```diff
@@ -31,6 +40,7 @@ func (ec *EppoClient) GetBoolAssignment(flagKey string, subjectKey string, subje
 	}
 	result, ok := variation.(bool)
 	if !ok {
+		ec.applicationLogger.Error("failed to cast %v to bool", variation)
```

Comment:
Does `Error` level feel appropriate here? It does to me since there is a type mismatch.